To: vim_dev@googlegroups.com
Subject: Patch 7.3.107
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.107
Problem:    Year number for :undolist can be confused with month or day.
Solution:   Change "%y" to "%Y".
Files:      src/undo.c


*** ../vim-7.3.106/src/undo.c	2011-01-22 00:11:42.000000000 +0100
--- src/undo.c	2011-01-22 00:03:58.000000000 +0100
***************
*** 2884,2890 ****
  	    (void)strftime((char *)buf, buflen, "%m/%d %H:%M:%S", curtime);
  	else
  	    /* long ago */
! 	    (void)strftime((char *)buf, buflen, "%y/%m/%d %H:%M:%S", curtime);
      }
      else
  #endif
--- 2884,2890 ----
  	    (void)strftime((char *)buf, buflen, "%m/%d %H:%M:%S", curtime);
  	else
  	    /* long ago */
! 	    (void)strftime((char *)buf, buflen, "%Y/%m/%d %H:%M:%S", curtime);
      }
      else
  #endif
*** ../vim-7.3.106/src/version.c	2011-01-22 21:05:02.000000000 +0100
--- src/version.c	2011-01-22 21:24:07.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     107,
  /**/

-- 
"The sun oozed over the horizon, shoved aside darkness, crept along the
greensward, and, with sickly fingers, pushed through the castle window,
revealing the pillaged princess, hand at throat, crown asunder, gaping
in frenzied horror at the sated, sodden amphibian lying beside her,
disbelieving the magnitude of the frog's deception, screaming madly,
"You lied!"
    - Winner of the Bulwer-Lytton contest (San Jose State University),
      wherein one writes only the first line of a bad novel

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
